style-guide/styles/wds-mixins/_button-theming.scss

```diff
@@ -0,0 +1,58 @@
+@import'../wds-functions/index';
+
+@mixin button-theming($color-primary, $color-primary-text, $color-secondary: "") {
+	$color-primary-active: lighten($color-primary, 20%);
+	@if luminance($color-primary) > .3 {
+		$color-primary-active: darken($color-primary, 20%);
+	}
+
+	@if $color-secondary == "" {
+		$color-secondary: $color-primary;
+	}
+
+	$color-secondary-active: lighten($color-secondary, 20%);
+	@if luminance($color-secondary) > .3 {
+		$color-secondary-active: darken($color-secondary, 20%);
+	}
+
+	background-color: $color-primary;
+	border-color: $color-primary;
+	color: $color-primary-text;
+
+	&:focus:not(:disabled),
+	&:hover:not(:disabled),
+	&:active,
+	&.wds-is-active {
+		background-color: $color-primary-active;
+		border-color: $color-primary-active;
+	}
+
+	&.wds-is-secondary {
+		background-color: transparent;
+		border-color: $color-secondary;
+		color: $color-secondary;
+
+		&:focus:not(:disabled),
+		&:hover:not(:disabled),
+		&:active,
+		&.wds-is-active {
+			background-color: transparent;
+			border-color: $color-secondary-active;
+			color: $color-secondary-active;
+		}
+	}
+
+	&.wds-is-text {
+		background-color: transparent;
+		border: 0;
```

Comment:
@igor I agree with border, however `background-color: transparent` I'd keep in the mixin to have all color-related properties in one place